ggu/main.c

```diff
@@ -151,20 +143,45 @@ process_single_packet(struct rte_mbuf *pkt, const struct ggu_config *ggu_conf)
 			goto free_packet;
 		}
 
-		udphdr = (struct udp_hdr *)&ip4hdr[1];
+		minimum_size +=
+			(ipv4_hdr_len(ip4hdr) - sizeof(*ip4hdr));
```

Comment:
Style: outmost parentheses on the right of `+=` are not needed; they neither improve readability nor are necessary.